spec/features/user_views_all_books_page_spec.rb

```diff
@@ -19,4 +19,19 @@
     expect(page).to have_css('.book', count: 2)
     expect(page).to have_css('.pagination')
   end
+
+  it "should have genre of books" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.